aws/eks/README.md

```diff
@@ -0,0 +1,24 @@
+# SecretHub + EKS
+
+This example uses Terraform to deploy the [SecretHub Demo App](https://secrethub.io/docs/start/getting-started/#consume) on a EKS cluster and provision it with the secrets it needs using the [SecretHub AWS Identity Provider](https://secrethub.io/docs/reference/aws/). 
+
+## Prerequisites
+1. [Terraform](https://www.terraform.io/downloads.html) installed along with the [SecretHub Provider](https://secrethub.io/docs/guides/terraform/#install)
+3. Correctly configured [AWS Credentials](https://www.terraform.io/docs/providers/aws/index.html#authentication)
+1. Variables in [variables.tf](./variables.tf)
```

Comment:
I think that could use some more explanation. Especially since the next section only says `terraform init` and `terraform apply`.

Reply:
Do you have a suggestion? I want to avoid introducing a sync-obligation between the README and `variables.tf`, so I'd prefer to keep the phrasing generic here.

By the way, applies to these READMEs too:
- [GCE](https://github.com/secrethub/examples/blob/285335c/google-cloud/gce/README.md#prerequisites)
- [GKE](https://github.com/secrethub/examples/blob/5fe1d85364d52e9df6e8f724bb9aa3e32ecf2931/google-cloud/gke/README.md#prerequisites)
- [EC2](https://github.com/secrethub/examples/blob/6ba364e5d787ec846dc20dff89f3e83fa7b0a82e/aws/ec2/README.md#prerequisites)

Reply:
What about?

> [Assign values](https://www.terraform.io/docs/configuration/variables.html#assigning-values-to-root-module-variables) to the variables in `variables`.

Reply:
`Values prepared for variables declared in [variables.tf](./variables.tf)`?

Reply:
I would honestly really like that link to the TF docs. Assigning values to a variable in a root module is not something you do on a daily basis.